scripts/gen_aac.py

```diff
@@ -200,46 +299,32 @@ def generate(self, download, jobs):
 
         with urllib.request.urlopen(self.url_reference_vectors_checksums) as resp:
             data = str(resp.read())
-            hparser.feed(data)
-        raw_bitstream_md5_links = [url for url in hparser.links if url.endswith(tuple(MD5_EXTS))]
-
+            hparser_checksums.feed(data)
+        raw_bitstream_md5_links = [
+            url for url in hparser_checksums.links if url.endswith(tuple(MD5_EXTS))
+        ]
         raw_bitstream_md5_names = [
-            os.path.splitext(os.path.splitext(os.path.basename(x))[0].split('_f')[0])[0] for x in
-            raw_bitstream_md5_links
+            os.path.splitext(os.path.splitext(os.path.basename(x))[0].split('_f')[0])[0] for x in raw_bitstream_md5_links
         ]
 
-        missing_checksum_files = [x for x in set(compressed_bitstream_names).difference(raw_bitstream_md5_names)]
+        missing_checksum_files = [
+            x for x in set(compressed_bitstream_names).difference(raw_bitstream_md5_names)
+        ]
         if missing_checksum_files:
-            print(f"Missing reference checksum files: {missing_checksum_files}")
+            #print(f"Missing reference checksum files: {missing_checksum_files}")
```

Comment:
Remove debug artifact

Reply:
This is old and unused code that I commented, but I forgot to remove it.